templates/data-sources/kubernetes_secret.md.tmpl

```diff
@@ -0,0 +1,23 @@
+---
+Title: "Secret Resource"
+Description: |-
+    Fetching the details of sceret resource.
+---
+
+# Namespace
+
+Manage a sceret using this Terraform module.
```

Comment:
Should there be some content difference between resource and data source, may be refer to the other docs and get it reviewed by @ishangupta-ds and the resource owners.

Reply:
@ishangupta-ds PTAL, at docs